ui/zenoedit/dock/zenodocktitlewidget.cpp

```diff
@@ -10,6 +10,7 @@
 #include "viewport/zenovis.h"
 #include "util/log.h"
 #include <QFileDialog>
+#include "../../../zenovis/xinxinoptix/xinxinoptixapi.h"
```

Comment:
```suggestion
#include "../../../zenovis/xinxinoptix/xinxinoptixapi.h"
```
Do you think every boy have OptiX? sir, we have non-CUDA build too, that's why the CI crashed.

Reply:
```suggestion
```